src/dbnode/environment/config.go

```diff
@@ -109,11 +98,9 @@ type ConfigureResults struct {
 
 // ConfigurationParameters are options used to create new ConfigureResults
 type ConfigurationParameters struct {
-	InstrumentOpts             instrument.Options
-	HashingSeed                uint32
-	HostID                     string
-	NamespaceResolutionTimeout time.Duration
```

Comment:
can't just delete these (b/w compatibility and all), need to mark deprecated instead

Reply:
Spoke offline and agreed its ok in this case because it was undocumented config and we're breaking the functionaltiy anyways